frontend/pages/hubs/[hubUrl].js

```diff
@@ -227,7 +227,7 @@ export default function Hub({
       {hubDescription && hubDescription.headContent && (
         <Head>{parseHtml(hubDescription.headContent)}</Head>
       )}
-      <WideLayout title={headline} fixedHeader headerBackground="#FFF">
+      <WideLayout title={headline} fixedHeader headerBackground="#FFF" image={getImageUrl(image)}>
```

Comment:
> One small problem I found additionally to the things noted in the code was that the share button overlaps with the "edit organization" button if you're on mobile and it's your own org.

For now, I just put the button higher up in this case. But maybe in the long run it would be worth relocating the edit organization button for tiny screens. It overlaps a bit with the organization image on very tiny screens. I can create an issue for that if wanted.

Reply:
Yes, it is always the hub image. Wouldn't I then need to pass the ideas image URL up from IdeaRoot.js to [HubUrl].js, which is not possible? Or is there a way to access the image URL of the currently opened idea from within [HubUrl].js? 
I also don't think I can just put a meta tag in IdeaRoot.js because it does not have an HTML head.

Reply:
Ah, I didn't check where the function to retrieve the idea was. Then it seems like it requires too much refactoring to be worth it so I'd just leave it as is :)